docs/source/python/api/compute.rst

```diff
@@ -555,3 +555,12 @@ Compute Options
    TrimOptions
    VarianceOptions
    WeekOptions
+
+Custom Functions
```

Comment:
```suggestion
User-Defined Functions
```

(to keep the titles consistent)

Reply:
@vibhatha Looks like you forgot to apply the suggestion before resolving?

Reply:
Oh sorry I missed this one.

Reply:
Added the change.